### PR TITLE
Add error handler for new search features

### PR DESCRIPTION
**Pull request by elsiehoffet-94**
This PR adds new error handlers for _include parameter. 
Also, if a resource is not stored in the database, the error will be logged but no OperationOutcome will be raised. (This prevents the _type, and _include functions from working properly).
A better error handling will be made with the refacto of the api and fhirstore. 

---
This change is [Reviewable](https://reviewable.io/reviews/arkhn/warehouse-api/51)

### CODE REVIEW

File: fhir-api/api.py [hunk 1]
```diff
@@ -129,7 +125,7 @@ def search_multiple_resources():
     search_args = {key: request.args.getlist(key) for key in request.args.keys()}
     if not search_args.get("_type"):
         raise OperationOutcome("No resource provided in _type parameter")
-    if "," not in search_args["_type"]:
+    if "," not in search_args["_type"][0]:
```
Comment (simonvadee) [inline fhir-api/api.py:128]: why did you add an index here ? isn't ` search_args["_type"]` a string ?

File: fhir-api/api.py [hunk 2]
```diff
@@ -152,12 +147,12 @@ def search_multiple_resources():
             result_per_resource = resource_search(
                 Model, processed_params, result_size, elements, offset, sort, include
             )
-            results["items"] += result_per_resource.get("items")
+            results["items"] += result_per_resource.get("items") if result_per_resource else {}
```
Comment (simonvadee) [inline fhir-api/api.py:150]: what about 
```
if not result_per_resource:
    continue
```
instead of all these ternaries ?

File: fhir-api/api.py (hunk 1 above)
Comment (elsiehoffet-94) [reply fhir-api/api.py:128]: No it's an array of strings here

File: fhir-api/api.py (hunk 2 above)
Comment (elsiehoffet-94) [reply fhir-api/api.py:150]: I changed it

File: fhir-api/api.py (hunk 1 above)
Comment (elsiehoffet-94) [reply fhir-api/api.py:128]: It comes from `request.args.getlist(key)`, which is used to deal with requests like Patient?name=John&name=Doe 
I'll probably think of a better way to do this in the refacto

File: fhir-api/api.py [hunk 3]
```diff
@@ -152,12 +147,13 @@ def search_multiple_resources():
             result_per_resource = resource_search(
                 Model, processed_params, result_size, elements, offset, sort, include
             )
+        if not result_per_resource:
+            continue
+        else :
```
Comment (Jasopaum) [inline fhir-api/api.py:152]: You don't need the `else` here, if you hit on `continue` you won't reach these lines

File: fhir-api/api.py (hunk 1 above)
Comment (Jasopaum) [reply fhir-api/api.py:128]: Hum, yeah, it's becoming hard to follow what's in each variable... Can't wait for the refactor! 🙂

Comment (Jasopaum) [reply fhir-api/api.py:128]: But did that change because you changed something else or was it wrong before?

Comment (elsiehoffet-94) [reply fhir-api/api.py:128]: It was wrong before :)

File: fhir-api/api.py (hunk 3 above)
Comment (elsiehoffet-94) [reply fhir-api/api.py]: Ok, I removed it!
